src/features/game/types/collectibles.ts

```diff
@@ -66,4 +67,13 @@ export const GOBLIN_BLACKSMITH_ITEMS: Record<
     supply: 535,
     boost: "+1 Blueberry",
   },
+  "Maneki Neko": {
+    description: "The beckoning cat. Pull it's arm and good luck will come",
+    ingredients: {
+      Gold: new Decimal(5),
+      "Red Envelope": new Decimal(50),
```

Comment:
This is just a placeholder right now - will need to design this mechanic + drop rate

Reply:
If its following the proposal, we can reuse the money tree modal to display the Red Evelope drop inside a chest reward.

Reply:
We've got a few technical constraints at the moment - the first one is that trees don't support random drop/reward mechanics yet 😢 We may need to reuse the crop reward system, unless we get time to focus on a tree drop system

Reply:
This don't support item rewards?

https://github.com/sunflower-land/sunflower-land/blob/5da412a6172d93d3f8aa5a564076203d50a95d5a/src/features/game/events/landExpansion/collectTreeReward.ts#L54

Reply:
My bad, I didn't even know we had that. Thank you to whoever implemented it 🙏